tests/test_hierarchy.cc

```diff
@@ -372,6 +382,27 @@ BOOST_DATA_TEST_CASE(
     ref_solution[std::make_tuple("hyper_ball" , "distort"    , "None"                  , "anasazi" , "matrix-free")] = 0.3029292535;
     ref_solution[std::make_tuple("hyper_ball" , "distort"    , "Reverse Cuthill_McKee" , "anasazi" , "matrix-full")] = 0.0970940159;
     ref_solution[std::make_tuple("hyper_ball" , "distort"    , "Reverse Cuthill_McKee" , "anasazi" , "matrix-free")] = 0.3029292535;
+
+    // FIXME Running ./test_hierarchy instead of ctest -R test_hierarchy gives a
+    // better convergence rate. Tightening the tolerance on anasazi-does reduces
```

Comment:
Why would `./test_hierarchy` and `ctest -R test_hierarchy` give different results? Don't they run the exact same executable with the same input data?

Reply:
Yeah I have no clue what's the difference.